sriovnet_switchdev.go

```diff
@@ -226,6 +226,44 @@ func findNetdevWithPortNameCriteria(criteria func(string) bool) (string, error)
 	return "", fmt.Errorf("no representor matched criteria")
 }
 
+// Finds the VF index of a port, if applicable
+func GetVfIndexFromNetDevice(netdev string) (int, error) {
+	if !isSwitchdev(netdev) {
+		return 0, fmt.Errorf("netdev %s is not an eswitch port", netdev)
+	}
+
+	physPortNameStr, err := getNetDevPhysPortName(netdev)
+	if err != nil {
+		return 0, fmt.Errorf("netdev %s has no physical backend", netdev)
```

Comment:
no need to hide underlying error.

`return 0, fmt.Errof("failed to get netdev %s physical port name. %v", netdev, err)`